pysensors/utils/_norm_calc.py

```diff
@@ -83,7 +84,7 @@ def max_n(lin_idx, dlens, piv, j, n_const_sensors, **kwargs):
         n_sensors = len(all_sensors)
     counter = 0
     # create a mask for constrained sensors in all sensors
-    # i.e., (all_sensors[mask[i]] == True means that sensor i is unconstrained
+    # i.e., (all_sensors[mask[i]] == True means that sensor i is unconstrained ##TODO: @Niha, should this be mask[all_sensors[i]] == True?
```

Comment:
Changed this to i.e., (mask[all_sensors[i]] == True means that sensor i is unconstrained